AndersenHWs/src/main/java/org/example/BusTicketValidator.java

```diff
@@ -0,0 +1,42 @@
+package org.example;
+
+import java.time.LocalDate;
+
+public class BusTicketValidator{
+    public static int startDateValidations = 0;
+    public static int ticketTypeValidations = 0;
+    public static int priceValidations = 0;
```

Comment:
1) If its static fields, so use UPPER_CASE
2) 0 its  default value for int, so just skip it, please